Lykke.Bil2.Bcn.TransactionsExecutor/src/Lykke.Bil2.Bcn.TransactionsExecutor/Services/TransactionExecutor.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Lykke.Bil2.Contract.Common;
+using Lykke.Bil2.Contract.TransactionsExecutor;
+using Lykke.Bil2.Contract.TransactionsExecutor.Requests;
+using Lykke.Bil2.Contract.TransactionsExecutor.Responses;
+using Lykke.Bil2.Sdk;
+using Lykke.Bil2.Sdk.Exceptions;
+using Lykke.Bil2.Sdk.TransactionsExecutor;
+using Lykke.Bil2.Sdk.TransactionsExecutor.Services;
+using Newtonsoft.Json;
+
+namespace Lykke.Bil2.Bcn.TransactionsExecutor.Services
+{
+    public class TransactionExecutor : ITransactionExecutor
+    {
+        public TransactionExecutor(/* TODO: Provide specific settings, if necessary */)
+        {
+        }
+
+        public async Task BroadcastAsync(BroadcastTransactionRequest request)
+        {
+            // TODO: broadcast transaction, throw TransactionBroadcastingException on error
```

Comment:
Let's add placeholder for request validation and description, that RequestValidationException should be thrown if request is invalid and can be never broadcasted

Reply:
Added exceptions description. Validation is blockchain specific, so I hardly could find an adequate example of validation. Actually there is an example of deserialization and throwing a RequestValidationException if deserialization failed.

Reply:
Yep, deserialization is good enough example